tests/runtime/rust_xcrate/world.wit

```diff
@@ -0,0 +1,26 @@
+package test:xcrate;
```

Comment:
Would it make sense to throw a resource type or two in here, just to test that cross-crate resource exports work as expected?